9MI0800355/oop_week1_adelina.cpp

```diff
@@ -0,0 +1,76 @@
+﻿#include <iostream>
+using namespace std;
+
+//ZAD 1
+//struct transition {
+//    int number;
+//    char ch;
+//};
+//
+//struct state{
+//    int data[4];
+//};
+//
+//struct stateInfo {
+//
+//};
+//int main()
+//{
+//    stateInfo;
+//}
+
+//int main() {
+//PairOfNumbers m;
+//m.first = 2;
+//m.second = 8;
+//print(m);
+//}
+
+
+//zad 2
+struct PairOfNumbers {
+	int first;
+	int second;
+};
+void print(const PairOfNumbers& arg) {
+	cout << arg.first << " " << arg.second << " ";
+}
+void input(PairOfNumbers& obj1) {
+	cin >> obj1.first >> obj1.second;
+}
+
+void compare(const PairOfNumbers& obj1, const PairOfNumbers& obj2) {
+	if ((obj1.first > obj1.second) || (obj1.first == obj1.second && obj2.first > obj2.second)) {

```

Comment:
Why not return a value instead ? The rule of thumb for writing comparators is as follows:
- return `negative integer` if `obj1 < obj2`
- return `zero` if `obj1 == obj2`
- return `positive integer` if `obj1 > obj2`